testdata/test.crai

```diff
@@ -0,0 +1 @@
+�;�C���&}�
```

Comment:
You might want to `testdata` -> `irobotclient/test/resources` and `testdir` ->  `irobotclient/test/resources` to keep all the test stuff together.